Content.Client/RoundEnd/RoundEndSummaryUIController.cs

```diff
@@ -33,14 +47,48 @@ private void ToggleScoreboardWindow(ICommonSession? session = null)
         }
     }
 
+    private void UnloadButton()
+    {
+        if (ToggleRoundEndSummaryWindowButton == null)
+            return;
+
+        ToggleRoundEndSummaryWindowButton.OnPressed -= ActionButtonPressed;
+    }
+
+    private void LoadButton()
+    {
+        if (ToggleRoundEndSummaryWindowButton == null)
+            return;
+
+        ToggleRoundEndSummaryWindowButton.OnPressed += ActionButtonPressed;
+    }
+
+    private void ActionButtonPressed(BaseButton.ButtonEventArgs args)
+    {
+        ToggleScoreboardWindow();
+    }
+
     public void OpenRoundEndSummaryWindow(RoundEndMessageEvent message)
     {
         // Don't open duplicate windows (mainly for replays).
         if (_window?.RoundId == message.RoundId)
             return;
 
+        if (ToggleRoundEndSummaryWindowButton != null)
+        {
+            ToggleRoundEndSummaryWindowButton.Pressed = true;
+            ToggleRoundEndSummaryWindowButton.Visible = true;
+        }
+
         _window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText,
             message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, EntityManager);
+
+        // When the RoundEndSummary Window closes, make sure to remove the "pressed" attribute when the windows closes
+        _window.OnClose += () =>
+        {
+            if (ToggleRoundEndSummaryWindowButton != null)
+                ToggleRoundEndSummaryWindowButton.Pressed = false;
+        };
     }
 
```

Comment:
Same here, the controller is being tightly coupled to a completely different window when the window itself is what should be handling it.